### PR TITLE
Fix: Region doesn't show face up (fixes #284 )

### LINKED ISSUE #284: Region doesn't show face up, when it has a content without values
**Describe the bug**
The region in the carbon-graphs does not show face up, even though there is only one line on the graph, if there is a content without values.

**Link to CodeSandbox**
Use [this](https://codesandbox.io/s/reverent-mendel-nphfx) template to reproduce the behavior.

**Steps to reproduce**
1. Load graph with a line graph with data containing region.
   * The line graph displays with region
2. Loan another line in the same graph/canvas which has empty array as values.
   * The region gets disappeared.

**Expected behavior**
The respective, region should appear where there is only one content in the graph.

**Screenshots**

**Desktop (please complete the following information):**

-   OS: [e.g. iOS]
-   Browser [e.g. chrome, safari]

**Additional context**
The legend without data will be either disabled/hidden. So user will not get a chance to unselect the legend. So the graph will think, it has more than one content and hides all the regions.

### PR DESCRIPTION
**Pull request by SharathAlur**
Fixes: https://github.com/cerner/carbon-graphs/issues/284

**What is the purpose of this pull request? (Check any that's applicable)**

-   [ ] Feature
-   [x] Bug fix
-   [ ] Dependency updates
-   [ ] Documentation updates
-   [ ] Build related changes
-   [ ] Changes an existing functionality
-   [ ] Other, please explain:

**Does this introduce a breaking change?**

-   [ ] Yes
-   [x] No

**What changes did you make? (Give an overview)**
Set `shouldHideAllRegion` to false if there is only one graph content displayed.
Check content's values count while confirming if a single target is displayed.

**Add any screenshots**
Screenshot 2020-09-09 at 3 19 17 PM
Screenshot 2020-09-09 at 3 20 25 PM

### CODE REVIEW

**Review by sdadn — APPROVED** [review]

**Comment by AshishMotanamGurunadham** [comment]
@SharathAlur Can you please add the the link to see these ui changes, so that we can add ux review required label.

**Comment by joelsamthomas** [comment]
When is this planned to be released? We are looking forward to consume these changes in our current sprint.

**Comment by Veera81** [comment]
@SharathAlur Can you please the dev site link in the PR description when you get a chance? So that we will add a UX review on this PR

File: src/main/js/controls/Line/helpers/helpers.js [hunk 1]
```diff
@@ -511,16 +511,17 @@ const drawDataPoints = (scale, config, pointGroupPath) => {
 };
 /**
  * Handler for Request animation frame, executes on resize.
- *  * Order of execution
- *      * Shows/hides the regions
+ * * Order of execution
+ * * Shows/hides the regions
```
Comment (AshishMotanamGurunadham) [inline src/main/js/controls/Line/helpers/helpers.js:515]: looks like we need gap between two stars in here to state that order of execution is show/hide the regions.

**Review by mraplumb — APPROVED** [review]
I am adding my approval to this PR in spite of my comments as they are not detrimental to the functionality being patched and in the interest of getting this patch released for consumers who need it.

**Review by jeremyfuksa — APPROVED** [review]

**Review by Veera81 — APPROVED** [review]

**Comment by Veera81** [comment]
@joelsamthomas  These changes are released in the version 2.13.4

**Pull request merged**